spec/src/consensus.rs

```diff
@@ -58,6 +59,13 @@ pub(crate) const MAX_BLOCK_CYCLES: u64 = TWO_IN_TWO_OUT_CYCLES * TWO_IN_TWO_OUT_
 const MAX_BLOCK_PROPOSALS_LIMIT: u64 = 3_000;
 const PROPOSER_REWARD_RATIO: Ratio = Ratio(4, 10);
 
+// A special lock for satoshi, use satoshi's private key can unlock this.
+pub(crate) const SATOSHI_LOCK_HASH: H256 =
+    h256!("0x1d11151e87cf8f77d2130e7ee8ca66be197b310ed39cc74c40a5df4933ca8a75");
```

Comment:
Better calculate from pubkey hash `0x62e907b15cbf27d5425399ebf6f0fb50ebb88f18` in a `lazy_static!`. If the serialization changed, it's invalid.

Reply:
This test guarantee the lock hash https://github.com/nervosnetwork/ckb/pull/1576/files/e49e1dec8527c7e6248aac2a035e327905b0628f#diff-ca7d4f96a4f6eae78978a2e2622a7c38R687

Reply:
But, if the serialization changed, you should manually update here. It's inconvenient and not straightforward.

Reply:
This value should seem as a constant, if serialization is changed, all the on-chain consensus are broken, we only need to manually handle this before mainnet launch.